wicket-console/src/main/java/ru/ydn/wicket/wicketconsole/HistoryItemPanel.html

```diff
@@ -8,6 +8,6 @@
 	<div class="wc-item-exception" wicket:id="exception"> </div>
 </div>
 <div class="wc-item-functions">
-	<div class="wc-item-clone" wicket:id="clone"> </div>
+	<a class="wc-item-reuse btn btn-default btn-xs" wicket:id="reuse"><i class="fa fa-repeat" aria-hidden="true"></i></a>
```

Comment:
Without fontawesome dependency you can just setup "some" CSS class and have some implemenetation there, but then in Orienteer overwrite CSS with what's needed.